init.rb

```diff
@@ -35,6 +35,18 @@
 Rock.flavors.alias 'stable', 'next'
 Rock.flavors.define 'master', :implicit => true
 
+def add_compiler_flag(varName, value)
+    curVal = Autobuild.env_value(varName)
```

Comment:
please stick to a ruby style. 

- no () around ifs
- variables are snake_case
- keyword parameters are `key: value`, not `:key => value` (that's "new")